source/azure/activity-services/prerequisites/index.rst

```diff
@@ -1,19 +1,20 @@
 .. Copyright (C) 2015, Wazuh, Inc.
 
 .. meta::
-  :description: Learn what you need to install and configure the Wazuh module to monitor Azure activity and services in this section of the Wazuh documentation. 
+  :description: Learn what you need to install and configure the Wazuh module to monitor Azure activity and services in this section of the Wazuh documentation.
 
 .. _azure_prerequisites:
 
 Prerequisites
 =============
 
-This module requires the right credentials to access the services. 
+This module requires dependencies as well as the right credentials to access the services.
```

Comment:
Install the dependencies required by this module and set the credentials to access the service. Check the `Considerations for configuration` section to learn more about configuring multiple services.